### ISSUE TITLE
Batch Normalization in nolearn

### ISSUE THREAD

**Issue by RishabGargeya** [dnouri/nolearn#208]
Hi,

I was just wondering how it would be possible to include batch normalization using nolearn.
Lasagne has a convenience function batch_norm, but I'm not sure how to use this in the format that you define neural nets using the nolearn wrapper?

**Comment by BenjaminBossan** [comment]
You can do that by initializing your layers lasagne style and then pass the last layer to the `layers` argument of `NeuralNet`. See [here](https://github.com/dnouri/nolearn/blob/master/nolearn/lasagne/tests/test_base.py#L507) for an example.

**Comment by dnouri** [comment]
Here's another example of using Lasagne-style layer initialization:

```python
layer = InputLayer(shape=(None, 1, 28, 28))
layer = Conv2DLayer(layer, filter_size=5, num_filters=8)
layer = MaxPool2DLayer(layer, pool_size=2)
layer = Conv2DLayer(layer, filter_size=5, num_filters=8)
layer = MaxPool2DLayer(layer, pool_size=2)
layer = DenseLayer(layer, num_units=128)
layer = DenseLayer(layer, nonlinearity=softmax, num_units=10)

net = NeuralNet(
    layer,
    update=nesterov_momentum,
    update_learning_rate=0.01,
    update_momentum=0.9,
    )
```
